featurebyte/worker/task/target_table.py

```diff
@@ -102,6 +106,7 @@ async def execute(self, payload: TargetTableTaskPayload) -> Any:
                 context_id=payload.context_id,
                 request_input=payload.request_input,
                 primary_entity_ids=primary_entity_ids,
+                purpose=purpose,
                 **additional_metadata,
             )
             await self.observation_table_service.create_document(observation_table)
```

Comment:
Could we write similar test like those in https://github.com/featurebyte/featurebyte/blob/main/tests/unit/worker/task/test_target_table.py?